locator/abstract_replication_strategy.hh

```diff
@@ -77,7 +77,7 @@ protected:
     replication_strategy_config_options _config_options;
     replication_strategy_type _my_type;
     bool _per_table = false;
-    bool _uses_tablets = false;
+    bool _uses_tablets;
```

Comment:
Can you mark it with `const` as well, so that compiler warns us if we forget to initialize it in some other constructor?

Reply:
Done.